doc/source/whatsnew/v0.23.1.txt

```diff
@@ -92,7 +92,7 @@ I/O
 
 - Bug in IO methods specifying ``compression='zip'`` which produced uncompressed zip archives (:issue:`17778`, :issue:`21144`)
 - Bug in :meth:`DataFrame.to_stata` which prevented exporting DataFrames to buffers and most file-like objects (:issue:`21041`)
--
+- Bug in IO JSON :func:`read_json`reading empty JSON schema with ``orient='table'`` back to :class:DataFrame caused an error (:issue:`21287`)
```

Comment:
@pyryjook : If you could address the merge conflicts, that would be great.

Reply:
Sure, I’ll certainly do that when I’m on my laptop again!